README.md

```diff
@@ -41,14 +41,16 @@ We release self-contained plug-and-play web components based on the [custom elem
 | [AXA Fieldset](./src/components/10-atoms/fieldset)                                 | [![npm version](https://img.shields.io/npm/v/@axa-ch/fieldset.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/fieldset)                             |
+| [AXA File Upload](./src/components/20-molecules/file-upload)                       | [![npm version](https://img.shields.io/npm/v/@axa-ch/file-upload.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/file-upload)                       |
 | [AXA Footer](./src/components/30-organisms/footer)                                 | [![npm version](https://img.shields.io/npm/v/@axa-ch/footer.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/footer)                                 |
 | [AXA Footer Small](./src/components/20-molecules/footer-small)                     | [![npm version](https://img.shields.io/npm/v/@axa-ch/footer-small.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/footer-small)                     |
 | [AXA Icon](./src/components/10-atoms/icon)                                         | [![npm version](https://img.shields.io/npm/v/@axa-ch/icon.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/icon)                                     |
 | [AXA Input File](./src/components/10-atoms/input-file)                             | [![npm version](https://img.shields.io/npm/v/@axa-ch/input-file.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/input-file)                         |
 | [AXA Input Text](./src/components/10-atoms/input-text)                             | [![npm version](https://img.shields.io/npm/v/@axa-ch/input-text.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/input-text)                         |
 | [AXA Link](./src/components/10-atoms/link)                                         | [![npm version](https://img.shields.io/npm/v/@axa-ch/link.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/link)                                     |
 | [AXA Materials](./src/components/00-materials)                                     | [![npm version](https://img.shields.io/npm/v/@axa-ch/materials.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/materials)                           |
-| [AXA Radio](./src/components/10-atoms/radio)                                     | [![npm version](https://img.shields.io/npm/v/@axa-ch/radio.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/radio)                           |
+| [AXA Radio](./src/components/10-atoms/radio)                                       | [![npm version](https://img.shields.io/npm/v/@axa-ch/radio.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/radio)                           |
+| [AXA Popup](./src/components/20-molecules/popup)                                   | [![npm version](https://img.shields.io/npm/v/@axa-ch/popup.svg?style=flat)](https://www.npmjs.com/package/@axa-ch/popup)                           |
```

Comment:
Popup wurde ausversehen released und muss unpublished werden.

Reply:
done in #1298